src/openvic-simulation/map/Province.cpp

```diff
@@ -34,9 +34,62 @@ Province::life_rating_t Province::get_life_rating() const {
 }
 
 bool Province::load_positions(BuildingManager const& building_manager, ast::NodeCPtr root) {
-	// TODO - implement province position loading
-	// (root is the dictionary after the province identifier)
-	return true;
+	return expect_dictionary_keys(
+		"text_position", ZERO_OR_ONE, [this](ast::NodeCPtr node) -> bool {
+			return expect_fvec2(assign_variable_callback(positions.text))(node);
+		},
```

Comment:
```suggestion
		"text_position", ZERO_OR_ONE, expect_fvec2(assign_variable_callback(positions.text)),
```
The `expect` functions are already callbacks, no need to wrap them in a lambda function (this applies to every `expect` function in this PR).